netebpfext/user/fwp_um.cpp

```diff
@@ -264,6 +266,7 @@ _fwp_engine::test_cgroup_inet4_connect(_In_ fwp_classify_parameters_t* parameter
     incoming_value[FWPS_FIELD_ALE_CONNECT_REDIRECT_V4_COMPARTMENT_ID].value.uint32 = parameters->compartment_id;
     incoming_value[FWPS_FIELD_ALE_CONNECT_REDIRECT_V4_ALE_APP_ID].value.byteBlob = &parameters->app_id;
     incoming_value[FWPS_FIELD_ALE_CONNECT_REDIRECT_V4_ALE_USER_ID].value.byteBlob = &parameters->user_id;
+    incoming_value[FWPS_FIELD_ALE_CONNECT_REDIRECT_V4_FLAGS].value.uint32=  FWP_CONDITION_FLAG_IS_REAUTHORIZE;
```

Comment:
I think `FWP_CONDITION_FLAG_IS_REAUTHORIZE` should be set in the incoming_value for `AUTH_CONNECT`.

Also we should not unconditionally set the flags here, as this will otherwise apply for each test case. All the test cases are not testing REAUTH case. We should have separate tests for REAUTH case.

Reply:
That is an excellent point. Thanks for explaining in detail. I modified and updated the test.